tendermint-rs/src/amino_types/vote.rs

```diff
@@ -174,7 +198,7 @@ mod tests {
         //		Round:            2,
         //		Timestamp:        stamp,
         //		Type:             byte(0x01), // pre-vote
-        //		BlockID: types.BlockID{
+        //		BlockId: types.BlockId{
```

Comment:
Nit: In golang this is actually called `BlockID`.

Reply:
Crap, forgot to fix this before merging, sorry 😢